pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
```

Comment:
I still think "to insert commands to be executed" is clearer/more accurate than "to execute commands". @ostera any thoughts?